src/modules/storage/LedgerStorage.ts

```diff
@@ -2057,6 +2115,23 @@ export class LedgerStorage extends Storages
 
         return this.query(sql, [tx_hash.toBinary(Endian.Little)]);
     }
+    /**
+     * Get Average Fees between given time range.
+     */
+    public calculateAvgFeeChart(from: number, to: number): Promise<any[]>{
+        let sql =
+            `SELECT
+                *
```

Comment:
```suggestion
                height, time_stamp, average_tx_fee
```
Looking up only the data you need in the chart can reduce the amount of data.